CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_fivetran_log v1.11.0
+[PR #141](https://github.com/fivetran/dbt_fivetran_log/pull/141) includes the following updates:
+
+## Schema Changes: Adding the transformation_runs table
+- We have added the `transformation_runs` source table. Note that not all customers have the `transformation_runs` source table, particularly if they are not using Fivetran Transformations. Therefore, the `transformation_runs` table will only populate if the table exists in your schema, via a new variable `fivetran_platform_using_transformations`, which automatically checks for the table. If the table doesn't exist, the staging `stg_fivetran_platform__transformation_runs` model will persist as an empty model and respective downstream fields will be null. 
+
+- If the `transformation_runs` source table exists in your schema, `fivetran_platform_using_transformations` will be set to True and the following updates apply:
+  - Added a new staging [`stg_fivetran_platform__transformation_runs`](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.stg_fivetran_platform__transformation_runs) model. 
+    - We have also added the `get_transformation_runs_columns()` macro to ensure all required columns are present.
+  - Added the following fields to the [`fivetran_platform__usage_mar_destination_history`](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_platform__usage_mar_destination_history) end model for each destination and month:
+    - `paid_model_runs`
+    - `free_model_runs`
+    - `total_model_runs`
+
+## Documentation Updates
```

Comment:
can you add the DECISIONLOG addition to this section and link to it

Reply:
done!